MbDotNet/Models/Stubs/HttpStub.cs

```diff
@@ -19,15 +19,23 @@ public class HttpStub : StubBase
         /// Adds a response to the stub that will return the specified HTTP status code.
         /// </summary>
         /// <param name="statusCode">The status code to be returned</param>
+        /// <param name="latencyInMilliseconds">The number of milliseconds to be waiting before status code will be returned</param>
         /// <returns>The stub that the response was added to</returns>
-        public HttpStub ReturnsStatus(HttpStatusCode statusCode)
+        public HttpStub ReturnsStatus(HttpStatusCode statusCode, int? latencyInMilliseconds = null)
```

Comment:
The general pattern on these stubs is that any of the helper methods like `ReturnsStatus(...)` or `OnStatusAndMethodEqual(...)` are to simplify the common use cases, while anything more complicated would be handled by the generic `Returns(ResponseBase)` and `On(PredicateBase)` methods which would allow the user to build their own custom responses/predicates.

I would prefer to avoid adding this parameter to each of the `ReturnsXYZ(...)` helpers and instead rely on the generic `Returns(...)` for responses with behaviors since I think the common case is to not include behaviors on your responses. I would be open to a discussion though if you feel adding latency to responses is common enough to warrant addition to these helper methods.